ideals.html

```diff
@@ -151,6 +151,12 @@
 	<p>Cockpit allows others to add additional <span class="caps">UI</span> pieces. We do
 	not allow arbitrary extension of the <span class="caps">UI</span> all over the place, but
 	at distinct extension&nbsp;points.</p>
+
+	<h2>Cockpit is&nbsp;tested</h2>
+	<p>Every feature and change in Cockpit is covered by an automatic test.
+	A proposed change only lands in the code base after it passes all
+	tests. Tests run on all Operating System releases on which Cockpit is
```

Comment:
I made "all OS" a link to http://cockpit-project.org/running.html, but we don't have an easily accessible per-release list (at most in our bots/tests-scan, but I don't think we want to link to that).